bin/node/runtime/src/lib.rs

```diff
@@ -91,6 +92,26 @@ pub type DealWithFees = SplitTwoWays<
 	Author, // 1 part (20%) goes to the block author.
 >;
 
+type SubmitPFTransaction =
+	frame_system::offchain::TransactionSubmitter<pallet_eth_offchain::crypto::Public, Runtime, UncheckedExtrinsic>;
+
+parameter_types! {
+   pub const BlockFetchDur: BlockNumber = 3;
+   // TODO: pass this from command line
+   pub const EthScanAPIKey: [u8; 34] = [74 as u8, 53 as u8, 51 as u8, 55 as u8, 68 as u8, 68 as u8,
```

Comment:
https://substrate.dev/docs/en/development/module/off-chain-workers#option-2-add-an-app-subkey-via-cli

Reply:
Thanks, I will find the solution from this document.